### PR TITLE
feat: support scss compile

### LINKED ISSUE #121: [feature] support embed sass
Integrated embed sass as plugin_sass to support compile `.scss`.

Investigate https://crates.io/crates/sass-embedded

### PR DESCRIPTION
**Pull request by shulandmimi**
**Description:**

support scss compile

**BREAKING CHANGE:**

`None`

**Related issue (if exists):**

### CODE REVIEW

File: js-plugins/sass/README.md
Comment (ErKeLost) [inline js-plugins/sass/README.md:1]: Readme should be in English.

Comment (ErKeLost) [inline js-plugins/sass/README.md:1]: Readme should be in English.

File: js-plugins/sass/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@farmfe/js-plugin-sass",
+  "version": "0.0.1",
+  "description": "support scss compile for farm.",
+  "main": "./dist/index.js",
```
Comment (wre232114) [inline js-plugins/sass/package.json:5]: define `exports`, `repository`, `bugs`, `author` field in package.json, and set `type="module"`.

File: js-plugins/sass/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "outDir": "dist",
+    "moduleResolution": "node"
```
Comment (wre232114) [inline js-plugins/sass/tsconfig.json:7]: remove this and use compile to native ESM

**Review by wre232114 — CHANGES_REQUESTED** [review]

**Comment by wre232114** [comment]
#121 should be implemented in Rust and using https://crates.io/crates/sass-embedded.

**Review by wre232114 — APPROVED** [review]

**Pull request merged**